backend/src/Types/Database.hs

```diff
@@ -334,3 +231,174 @@ updatedPlayerStatsLose side rating (PlayerStatsTotal {..}) (PlayerStatsYear {..}
           ..
         }
     )
+
+instance ToNamedRecord Player where
+  toNamedRecord :: Player -> CSV.NamedRecord
+  toNamedRecord Player {..} =
+    CSV.namedRecord
+      [ "name" .= playerName,
+        "display_name" .= playerDisplayName,
+        "country" .= fromMaybe ("" :: Text) playerCountry,
+        "is_active" .= if playerIsActive then ("Active" :: Text) else "Inactive"
+      ]
+
+playerCsvHeader :: Header
+playerCsvHeader = CSV.header ["name", "display_name", "country", "is_active"]
+
+data ExportGameReport = ExportGameReport
+  { exportGameReportRecord :: GameReport,
+    exportGameReportWinnerName :: PlayerName,
+    exportGameReportLoserName :: PlayerName
+  }
+
+instance ToNamedRecord ExportGameReport where
+  toNamedRecord :: ExportGameReport -> CSV.NamedRecord
+  toNamedRecord
+    ExportGameReport
+      { exportGameReportRecord =
+          GameReport {..},
+        ..
+      } =
+      CSV.namedRecord
+        [ "timestamp" .= T.show gameReportTimestamp,
+          "winner_name" .= exportGameReportWinnerName,
+          "loser_name" .= exportGameReportLoserName,
+          "side" .= T.show gameReportSide,
+          "victory" .= T.show gameReportVictory,
+          "match" .= T.show gameReportMatch,
+          "competition" .= T.intercalate "," (map show gameReportCompetition),
+          "league" .= maybe ("" :: Text) show gameReportLeague,
+          "expansions" .= T.intercalate "," (map show gameReportExpansions),
+          "treebeard" .= maybe ("" :: Text) (\b -> if b then "true" else "false") gameReportTreebeard,
```

Comment:
Nit: similar to how you mapped player status from true/false to active/inactive, this could be an opportunity to map to mustered/not mustered?